### PR TITLE
Kdepim meta

### PR DESCRIPTION
**Pull request by NeddySeagoon**
kdepim-meta for ~arm64 users. kdepim-meta mail-filter/spamassassin is in package.use.mask meanwhile.
Its known to build and install with FEATURES=test USE=test but it real life is picking up, so I'll keyword it later.

### CODE REVIEW

File: profiles/arch/arm64/package.use.mask [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # Copyright 1999-2019 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
+# Roy Bamford <neddyseagoon@gentoo.org> (12 Mar 2019)
+# Let kde-apps/akonadi use mysql.
```
Comment (a17r) [inline profiles/arch/arm64/package.use.mask]: This comment seems backwards - why not look at existing examples and say `dev-db/mysql not keyworded yet` there.

Comment (NeddySeagoon) [reply profiles/arch/arm64/package.use.mask]: How about I change the comment to read
# Allow kde-apps/akonadi to use virtual/mysql 
which is what it really means for the akonadi ebuild.

dev-db/mysql is neither keyworded on arm64 nor required to satisfy virtual/mysql 
arm64 users will actually get dev-db/mariadb

Comment (NeddySeagoon) [reply profiles/arch/arm64/package.use.mask]: Heh, leading hash symbols

Comment (a17r) [reply profiles/arch/arm64/package.use.mask]: But what it really means is `allow kde-apps/akonadi to use postgresql or sqlite` while mysql is not available.

Comment (a17r) [reply profiles/arch/arm64/package.use.mask]: Ha. No I get it.

**Pull request closed without merging**